src/sentry/static/sentry/app/views/alerts/details/header.tsx

```diff
@@ -69,89 +66,123 @@ export default class DetailsHeader extends React.Component<Props> {
+            <SubscribeButton
+              disabled={!isIncidentReady}
+              isSubscribed={incident?.isSubscribed}
+              onClick={onSubscriptionChange}
+              size="small"
+            />
+            {this.renderStatus()}
+          </Controls>
+        </BreadCrumbBar>
+        <Details>
+          <div>
+            <IncidentTitle data-test-id="incident-title" loading={!isIncidentReady}>
+              {incident && !hasIncidentDetailsError ? incident.title : 'Loading'}
+            </IncidentTitle>
+            <IncidentSubTitle loading={!isIncidentReady}>
+              Triggered: {dateStarted}
+            </IncidentSubTitle>
+          </div>
+
+          {hasIncidentDetailsError ? (
+            <StyledLoadingError />
+          ) : (
+            <GroupedHeaderItems columns={isErrorDataset ? 5 : 3}>
+              <ItemTitle>{t('Environment')}</ItemTitle>
+              <ItemTitle>{t('Project')}</ItemTitle>
+              {isErrorDataset && stats && <ItemTitle>{t('Users affected')}</ItemTitle>}
+              {isErrorDataset && stats && <ItemTitle>{t('Total events')}</ItemTitle>}
+              <ItemTitle>{t('Active For')}</ItemTitle>
+              <ItemValue>{environmentLabel}</ItemValue>
               <ItemValue>
-                <Duration seconds={getDynamicText({value: duration || 0, fixed: 1200})} />
+                {project && (
+                  <Projects slugs={[project]} orgId={params.orgId}>
+                    {({projects}) => (
+                      <ProjectBadge
+                        avatarSize={18}
+                        project={projects && projects.length && projects[0]}
+                      />
+                    )}
+                  </Projects>
+                )}
               </ItemValue>
-            )}
-            <ItemValue>
-              <SubscribeButton
-                disabled={!isIncidentReady}
-                isSubscribed={incident && !!incident.isSubscribed}
-                onClick={onSubscriptionChange}
-                size="small"
-              />
-            </ItemValue>
-          </GroupedHeaderItems>
-        )}
+              {isErrorDataset && stats && (
+                <ItemValue>
+                  <Count value={stats.uniqueUsers} />
+                </ItemValue>
+              )}
+              {isErrorDataset && stats && (
+                <ItemValue>
+                  <Count value={stats.totalEvents} />
+                </ItemValue>
+              )}
+              {incident && (
+                <ItemValue>
+                  <Duration
+                    seconds={getDynamicText({value: duration || 0, fixed: 1200})}
+                  />
+                </ItemValue>
+              )}
+            </GroupedHeaderItems>
+          )}
+        </Details>
       </Header>
     );
   }
 }
 
-const Header = styled(PageHeader)`
-  background-color: ${p => p.theme.white};
+const Header = styled('div')`
+  background-color: ${p => p.theme.offWhite};
```

Comment:
@robinrendle just updated our theme, I believe there's a new color value we should be using

Reply:
Yep – you can switch that out for `gray100` now